ipa-core/src/protocol/prss/mod.rs

```diff
@@ -232,7 +203,7 @@ impl RngCore for SequentialSharedRandomness {
     // That is OK for the same reason that we use in converting a `u128` to a small `Field`.
     #[allow(clippy::cast_possible_truncation)]
     fn next_u64(&mut self) -> u64 {
-        let v = self.generator.generate(self.counter);
+        let v = self.generator.generate(self.counter.offset(0));
```

Comment:
I'm not sure what's going on here and how this is connected to the PR.

Reply:
This change is required now because `generate` method takes `PrssIndex128` type instead of `u128`. It does feel that for consistency reasons we should use `PrssIndex`/`PrssIndex128` everywhere in PRSS module, but maybe @andyleiserson has different opinion about it